src/components/EditorBottomBar.tsx

```diff
@@ -1,15 +1,32 @@
 import "./EditorBottomBar.css";
 import { FormattedMessage } from "react-intl";
+import { useAppServices } from "./AppServicesContext";
 
-interface EditorBottomBarProps {}
+interface EditorBottomBarProps {
+  nextSrc: string;
+}
+
+export const EditorBottomBar = ({
+  nextSrc,
+  ...otherProps
+}: EditorBottomBarProps) => {
+  const {
+    window: { location },
+  } = useAppServices();
+  const onContinue = (): any => {
+    location.href = nextSrc;
+  };
```

Comment:
Creo que este código va a ser removido, así que solo hago esta sugerencia para casos futuros.

En general, conviene evitar anotaciones de tipos que salen del contexto. Y en particular, conviene evitarlas cuando no agregan información, como `any` en este caso.

```suggestion
  const onContinue = () => {
    location.href = nextSrc;
  };
```

Reply:
Relacionado: https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html